Zeeguu Reader/Zeeguu Reader/ArticleListViewController.swift

```diff
@@ -116,26 +116,17 @@ class ArticleListViewController: ZGTableViewController {
 	}
 
 	override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell")
-		var cell: UITableViewCell
+		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell") as? ArticleTableViewCell
```

Comment:
Thats indeed the case. However, this is also in the Apple docs. Do I really need to cite the Apple docs in cases like this?

https://developer.apple.com/library/prerelease/ios/documentation/UIKit/Reference/UITableView_Class/index.html#//apple_ref/occ/instm/UITableView/dequeueReusableCellWithIdentifier: